readme.txt

```diff
@@ -73,6 +73,50 @@ Start editing inside the `onoffice-personalized` folder.
 
 == Changelog ==
 
+= 2.20.11 =
```

Comment:
Shouldn't this have been merged during the merge or rebase?

Reply:
@jayay sorry, we do not understand your comment, the change is not included in the PR

Reply:
Oh, maybe GitHub included it for some reason then.